lms/djangoapps/grades/course_grades.py

```diff
@@ -14,7 +14,7 @@
 log = getLogger(__name__)
 
 
-GradeResult = namedtuple('StudentGrade', ['student', 'gradeset', 'err_msg'])
+GradeResult = namedtuple('GradeResult', ['student', 'gradeset', 'err_msg'])
```

Comment:
?  How did this even work before?  I thought using the named tuple would have complained about it.